tools/windows-node-installer/cmd/aws.go

```diff
@@ -35,7 +35,7 @@ func init() {
 func newAWSCmd() *cobra.Command {
 	awsCmd := &cobra.Command{
 		Use:   "aws",
-		Short: "Takes aws specific resource names from user",
+		Short: "Create and destroy windows instances in aws",
```

Comment:
@Bowenislandsong  this is the change I did specific to aws, and also in the README as well.

Reply:
For future reference, this should be a separate PR

Reply:
Ok, thank you.